### PR TITLE
bundles script

### PR DESCRIPTION
**Pull request by adwk67**
This replaces the operator-based scripts. The manifests used for creating PRs against the RedHat repository can now be used for creating bundles and catalogs for testing prior to certification pipeline runs.

### CODE REVIEW

File: olm/build-bundles.sh [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+}
+
+main() {
+  parse_inputs "$@"
+  if [ -z "${VERSION}" ] || [ -z "${BRANCH}" ] || [ -z "${OPERATOR_NAME}" ]; then
+    echo "Usage: $SCRIPT_NAME -r <release> -b <branch> -o <operator>"
+    exit 1
+  fi
+
+  rm -rf "/tmp/openshift-bundles"
```
Comment (sbernauer) [inline olm/build-bundles.sh:126]: Works for me. Just for future projects you could use something like `mktemp -d -t 'openshift-bundles-XXXXXXXX'`, which gives e.g. `/tmp/openshift-bundles-eUCh4ns8`.

File: olm/templates/workspace-template.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (sbernauer) [inline olm/templates/workspace-template.yml:1]: Is this file needed?

File: olm/build-bundles.sh [hunk 3]
```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+# This script does the following:
+#   - clones
```
Comment (sbernauer) [inline olm/build-bundles.sh:3]: Guess this sentence is incomplete

File: olm/build-bundles.sh [hunk 4]
```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
```
Comment (sbernauer) [inline olm/build-bundles.sh:1]: Some documentation such as

You need to have the following tools installed
* olm, see https://github.com/operator-framework/operator-lifecycle-manager/releases/latest

would be great, as well as a mention that you kube-context must point at an OS cluster

**Review by sbernauer — CHANGES_REQUESTED** [review]

File: olm/build-bundles.sh (hunk 1 above)
Comment (adwk67) [reply olm/build-bundles.sh:126]: Thanks! Will incorporate that.

File: olm/templates/workspace-template.yml (hunk 2 above)
Comment (adwk67) [reply olm/templates/workspace-template.yml:1]: Yes, it's needed for the CI pipeline callouts (subject of the TODO mentioned above)

File: olm/build-bundles.sh (hunk 4 above)
Comment (adwk67) [reply olm/build-bundles.sh:1]: Done.

**Review by sbernauer — APPROVED** [review]
Many thanks!
